internal/pkg/deploy/cloudformation/stack/testdata/workloads/svc-manifest.yml

```diff
@@ -68,6 +68,12 @@ environments:
         path: /
         grace_period: 30s
       deregistration_delay: 30s
+    secrets:
```

Comment:
qq: do we add this test to both `staging` and `prod` out of some specific concerns?

Reply:
When the manifest contains 
```yaml
logging
  secrets:
    <key>: <value>
```
This will expose Secrets to the sidecar container, which I am testing in `prod` environment.

When the manifest contains
```yaml
secrets:
  <key>:  <value>
``` 
Secrets will be securely passed to your service as environment variables, which I am testing in `staging` environment.
Yes you are correct. I can test both of these in the same environment.